src/ccs/multifolding.rs

```diff
@@ -101,55 +149,42 @@ impl<G: Group> Multifolding<G> {
 
   /// Compute g(x) polynomial for the given inputs.
   pub fn compute_g(
-    running_instance: &LCCCS<G>,
-    cccs_instance: &CCCSShape<G>,
-    z1: &Vec<G::Scalar>,
-    z2: &Vec<G::Scalar>,
+    &self,
+    cccs_instance: &CCCS<G>,
```

Comment:
Here is called `cccs_instance`, but in `fold` & `fold_z` it is called `cccs` (in all cases being a `CCCS<G>` struct. Maybe we could unify the variable names?

Reply:
Fixed in e52e174e